readme.md

```diff
@@ -1,88 +1,183 @@
+    fpAgent.get().then(result => {
+      // This is the visitor identifier:
+      const visitorId = result.visitorId;
+      console.log(visitorId);
+    });
   }
 </script>
 <script
-  async src="https://cdn.jsdelivr.net/npm/@fingerprintjs/fingerprintjs@3.0.0-beta.2/dist/fp.min.js"
+  async src="https://cdn.jsdelivr.net/npm/@fingerprintjs/fingerprintjs@3/dist/fp.min.js"
   onload="FingerprintJS.load().then(onFingerprintJSLoad)"
 ></script>
 ```
 
-A simple way to start but not recommended because AdBlock and similar browser extensions often block this URL.
-You should at least upload the script to your server.
+This is a simple way to start but not recommended because AdBlock and other browser extensions can block the script.
+You should at least upload the script file to your server.
 
-### Webpack/Rollup/Browserify
+### Or from NPM and use with Webpack/Rollup/Browserify
 
 ```bash
 npm i @fingerprintjs/fingerprintjs
+# or
+yarn add @fingerprintjs/fingerprintjs
 ```
 
 ```js
-import * as FPJS from '@fingerprintjs/fingerprintjs';
+import * as FingerprintJS from '@fingerprintjs/fingerprintjs';
 
 (async () => {
-  const fpjs = await FPJS.load();
-  const { visitorId } = await fpjs.get();
-  console.log(visitorId)
+  // It's good to call this when the application starts
+  const fpAgent = await FingerprintJS.load();
+
+  // The FingerprintJS agent is ready. Get a visitor identifier when you'd like to.
+  const result = await fpAgent.get();
+
+  // This is the visitor identifier:
+  const visitorId = result.visitorId;
+  console.log(visitorId);
 })();
 ```
 
-## Version policy
+## Pro version
 
-The library doesn't promise same visitor identifier between any versions
-but tries to keep them same as much as reasonable.
+Upgrade to [Pro version](https://fingerprintjs.com) to get 99.5% accuracy of identification.
 
-The documented JS API follows [Semantic Versioning](https://semver.org).
-Using undocumented features is at you own risk.
+<p align="center">
+  <a href="https://fingerprintjs.com">
+    <img src="resources/pro_screenshot.png" alt="Pro screenshot" width="697px" />
+  </a>
+</p>
 
-## Contribution
+| | Open Source version | Pro version |
+|-----|-------|-------|
+| Browser identification | Basic | Advanced |
+| Anonymous user identification | ❌ | ✅ |
+| Bot detection | ❌ | ✅ |
+| Incognito / Private mode detection | ❌ | ✅ |
+| Geolocation | ❌ | ✅ |
+| Security | ❌ | ✅ |
+| Server API | ❌ | ✅ |
+| Webhooks | ❌ | ✅ |
 
-### Development playground
+Pro result example:
 
-```bash
-# Make sure you have Yarn installed
-yarn install
-yarn playground:start # Add '--port 8765' to change the server port
+```js
+{
+  "requestId": "HFMlljrzKEiZmhUNDx7Z",
+  "visitorId": "kHqPGWS1Mj18sZFsP8Wl",
+  "visitorFound": true,
+  "incognito": false,
+  "bot": undefined,
+  "ip": "192.65.67.131",
+  "browserName": "Chrome",
+  "browserVersion": "85.0.4183",
+  "os": "Mac OS X",
+  "osVersion": "10.15.6",
+  "device": "Other",
+  "ipLocation": {/* ... */}
```

Comment:
can we have inline lat and lng attributes there ?

Reply:
Only them? There are many attributes hidden here